libtelemetry/src/test/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -25,6 +30,14 @@
 
 public class MapboxTelemetryTest {
 
+  SharedPreferences mockedPreferences = Mockito.mock(SharedPreferences.class);
```

Comment:
This is not needed. Adding `RETURNS_DEEP_STUBS` to the mocked `Context` in the tests in which `SharedPreferences` are involved and ensuring that `isTelemetryEnabled` has the proper value in each case should be enough.